packages/core/src/modules/connections/messages/ConnectionInvitationMessage.ts

```diff
@@ -127,9 +126,7 @@ export class ConnectionInvitationMessage extends AgentMessage {
 
     if (typeof encodedInvitation === 'string') {
       const invitationJson = JsonEncoder.fromBase64(encodedInvitation)
-      const invitation = JsonTransformer.fromJSON(invitationJson, ConnectionInvitationMessage)
-
-      await MessageValidator.validate(invitation)
+      const invitation = JsonTransformer.fromJSON(invitationJson, ConnectionInvitationMessage, { validate: false })
```

Comment:
This was validating before and now it has validate false. Can we keep validation enabled here?

Reply:
Also, I think we can make this method sync now!